SU-SWS/stanford_components
# Earth 375 video hero

# READY FOR REVIEW

# Summary
- Changed hero banner pattern to render the image instead of using background image
- Added styling for video compatibility

# Needed By (Date)
- End of Sprint

# Urgency
- Medium (they really want it)

# Steps to Test

1. Checkout this branch and same branch in [stanford_paragraph_types](https://github.com/SU-SWS/stanford_paragraph_types/pull/88)
2. clear cache & `features-import stanford_paragraph_video stanford_paragraph_hero_banner -y`
3. Review home page hero banner and any other hero banners look normal still and that it contains a `<picture>` tag now instead of background image
4. Create complex page with video type
5. Set the display variant to full width
6. verify the video is playable and the text overlays with a hairline (maybe a style change will change this for non-full width display? )
7. Also feel free to review patterns page for the hero_banner @ `patterns/hero_banner`

# Associated Issues and/or People
- https://github.com/SU-SWS/stanford_paragraph_types/pull/88

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
